bin/logdaemon.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
```

Comment:
ok, then I got this wrong, I believed a python2 binary was always present since python 2.6
```

easybuild-framework (system_config) $ which python
/Library/Frameworks/Python.framework/Versions/2.7/bin/python
easybuild-framework (system_config) $ which python2
/Library/Frameworks/Python.framework/Versions/2.7/bin/python2
```